### PR TITLE
LCH-2721 | Rename 'CDF' to 'Cdf'.

### PR DESCRIPTION
**Pull request by ynx**
In LCH-2721, I have made those changes but somehow ended up not committing them. I am just trying to get them in now.

### CODE REVIEW

File: test/CDFDocumentTest.php [hunk 1]
```diff
@@ -145,7 +145,7 @@ public function testMergeDocumentsByKeys($setOne, $setTwo)
    */
   public function testMergeDocumentsNoOverlap($setOne, $setTwo, $elementFromSetTwo)
   {
-    $this->cdfDocument->setCDFEntities(...$setOne);
+    $this->cdfDocument->setCdfEntities(...$setOne);
     $documentToMerge = new CDFDocument(...$setTwo);
```
Comment (c0d39uru) [inline test/CDFDocumentTest.php:149]: And CDFDocument?

File: test/CDFDocumentTest.php [hunk 2]
```diff
@@ -65,7 +65,7 @@ public function testGetEntity($objectToAdd, $notAddedObject)
   public function testGetEntities($objectOne, $objectTwo)
   {
     //Check getting added and not added Object
-    $this->cdfDocument->setCDFEntities($objectOne, $objectTwo);
+    $this->cdfDocument->setCdfEntities($objectOne, $objectTwo);
 
     foreach ($this->cdfDocument->getEntities() as $entity) {
       $this->assertInstanceOf(CDFObject::class, $entity);
```
Comment (c0d39uru) [inline test/CDFDocumentTest.php:71]: Should CDFObject also be changed?

File: src/CDFDocument.php [hunk 3]
```diff
@@ -27,7 +27,7 @@ class CDFDocument
      */
     public function __construct(CDFObject ...$entities)
```
Comment (c0d39uru) [inline src/CDFDocument.php:28]: Should CDFObject also be changed?

File: src/EventSubscriber/DefaultCDF.php [hunk 4]
```diff
@@ -10,7 +10,7 @@
 /**
  * The Configuration entity CDF creator.
  *
- * @see \Drupal\acquia_contenthub\Event\CreateCDFEntityEvent
+ * @see \Drupal\acquia_contenthub\Event\CreateCdfEntityEvent
  */
 class DefaultCDF implements EventSubscriberInterface
```
Comment (c0d39uru) [inline src/EventSubscriber/DefaultCDF.php:15]: And DefaultCDF?

**Review by c0d39uru — APPROVED** [review]

**Comment by ynx** [comment]
@looking-4-justice, I tried to update the class and file names to change CDF to Cdf, but [because the composer.json is pinned to content-hub-php repo's 2.0.0-beta1 tage](https://travis-ci.com/acquia/content-hub-d8/jobs/192526763#L1225), the tests are failing. I can still try to change this but I think it's a lot of trouble. I am going to stick with changing only the function names this time.

File: test/CDFDocumentTest.php (hunk 1 above)
Comment (ynx) [reply test/CDFDocumentTest.php:149]: Out of scope. Please see my overall comment on this PR.

File: test/CDFDocumentTest.php (hunk 2 above)
Comment (ynx) [reply test/CDFDocumentTest.php:71]: Out of scope. Please see my overall comment on this PR.

File: src/EventSubscriber/DefaultCDF.php (hunk 4 above)
Comment (ynx) [reply src/EventSubscriber/DefaultCDF.php:15]: Out of scope. Please see my overall comment on this PR.

**Pull request merged**